src/test/ui/arbitrary-self-types-not-object-safe.rs

```diff
@@ -12,38 +12,38 @@
 use std::rc::Rc;
 
 trait Foo {
```

Comment:
I feel like we should have some positive tests too, no? 

e.g., some tests where you invoke a `Rc<Self>` method through a `Rc<dyn Trait>` value?

Reply:
well I guess the trans parts aren't implemented, but it'd be nice to at least check that the trait is considered object safe...

Reply:
you can use `#[rustc_error]` perhaps to error out before trans executes =)

Reply:
I added a new test – src/test/compile-fail/arbtrary-self-types-pointers-and-wrappers.rs, which uses `#[rustc_error]` for now, and will be converted to a run-pass test once dynamic dispatch is implemented